utils/src/main/java/io/confluent/common/utils/Utils.java

```diff
@@ -356,4 +363,34 @@ public static Properties loadProps(String filename) throws IOException {
     return props;
   }
 
+  /**
+   * Recursively delete the given file/directory and any subfiles (if any exist)
+   *
+   * @param file The root file at which to begin deleting
+   */
+  public static void delete(final File file) throws IOException {
+    if (file == null)
+      return;
+    Files.walkFileTree(file.toPath(), new SimpleFileVisitor<Path>() {
+      @Override
+      public FileVisitResult visitFileFailed(Path path, IOException exc) throws IOException {
+        // If the root path did not exist, ignore the error; otherwise throw it.
+        if (exc instanceof NoSuchFileException && path.toFile().equals(file))
```

Comment:
you need `{}` on this if statement

Reply:
We have a bunch of stuff copied from AK that we have ignored checkstyle on. This is sort of that, though I didn't copy all the variants/the complete flexibility of the part I did copy.

I've adjusted to match style, but we should probably either decide to abandon the AK compatibility or avoid making more formatting changes in the future.